op-challenger/game/fault/trace/vm/executor.go

```diff
@@ -57,6 +58,7 @@ type Config struct {
 	Networks          []string
 	L2Custom          bool
 	RollupConfigPaths []string
+	L1GenesisPath     string
```

Comment:
I don't actually see this value getting set in production - looks like we're only setting this up in tests?  For example, we create new vm configs [here](https://github.com/ethereum-optimism/optimism/blob/gk/op-node-l1-chain-config/op-challenger/flags/flags.go#L710-L710), [here](https://github.com/ethereum-optimism/optimism/blob/gk/op-node-l1-chain-config/op-challenger/config/config.go#L223), and [here](https://github.com/ethereum-optimism/optimism/blob/gk/op-node-l1-chain-config/op-challenger/config/config.go#L153).

Reply:
[0e14f8c](https://github.com/ethereum-optimism/optimism/pull/17568/commits/0e14f8c1bfdc66f17698b4a09a259d99148e17f2)

Reply:
There's still some vm config setup in [this file](https://github.com/ethereum-optimism/optimism/blob/gk/op-node-l1-chain-config/op-challenger/config/config.go) which looks like gets wired through to e2e tests.  So, I think we might start getting test failures when we update our test setup to include the latest l1 hardfork.  I think it would probably be fine to log a TODO and come back to this though.